.travis.yml

```
@@ -82,6 +82,11 @@ script:
     if [[ -v MAVEN_CHECKS ]]; then
       ./mvnw install -DskipTests -B -T C1 -P ci
     fi
+  - |
+    if [[ -v MAVEN_CHECKS ]]; then
```

I realized my suggestion (https://github.com/prestosql/presto/pull/590#discussion_r272581970) was not good enough. You should be able to go with:

```suggestion
    if [[ -v BUILD_PRESTO_DOCKER ]]; then
```

\+ replace 
https://github.com/prestosql/presto/blob/d647209e83464c9a63bd99eea5e2e71a0c8d13c6/.travis.yml#L12
with
```
- MAVEN_CHECKS=true BUILD_PRESTO_DOCKER=true
```

---

👍